src/utils/solanaHelpers.ts

```diff
@@ -300,12 +332,12 @@ export async function getEstimateBalanceChange(connection: Connection, tx: Trans
       return prev;
     }, new Map<string, PublicKey>());
 
-    log.info(tx instanceof Transaction);
     // add selected Account incase signer is just fee payer (instruction will not track fee payer)
     accounts.set(signer, new PublicKey(signer));
 
     // Simulate Transaction with Accounts
-    const result = await connection.simulateTransaction(tx.compileMessage(), undefined, Array.from(accounts.values()));
+    const addresses = Array.from(accounts.keys());
```

Comment:
Need to take account for V0 transaction which have ATL

Reply:
have fixed but should add parser for address lookup table transaction in display activities related places